styles/pypi/catppuccin.user.css

```diff
@@ -0,0 +1,1080 @@
+    }
+
+    .site-header {
+      background-color: @crust !important;
+      border-bottom-width: 0 !important;
+    }
+
+    .site-header__logo:active,
+    .site-header__logo:focus {
+      outline-color: @text !important;
+    }
+
+    .dropdown__content {
+      border: 1px solid @accent-color !important;
+      box-shadow: none !important;
+    }
+
+    .dropdown__content li:hover {
+      background-color: @base !important;
+    }
+
+    .dropdown button.dropdown__link:not(:hover),
+    .dropdown__link {
+      background-color: @crust !important;
+      border-bottom-width: 0 !important;
+      color: @text !important;
+    }
+
+    .dropdown__link:hover {
+      background-color: @base !important;
+    }
+
+    a.dropdown__link,
+    a.dropdown__link:hover {
+      background-image: none !important;
+      color: @text !important;
+    }
+
+    .horizontal-menu__link:active,
+    .horizontal-menu__link:focus {
+      outline-color: @accent-color !important;
+    }
+
+    .horizontal-menu--light .horizontal-menu__link {
+      color: @accent-color !important;
+    }
+
+    .horizontal-menu--light .horizontal-menu__link:hover {
+      text-decoration-color: @accent-color !important;
+    }
+
+    .package-snippet {
+      background-color: @crust !important;
+      border: none !important;
+      box-shadow: none !important;
+    }
+
+    .package-snippet__created {
+      color: @subtext0 !important;
+    }
+
+    .package-snippet__description {
+      color: @subtext1 !important;
+    }
+
+    .button {
+      border-color: @subtext0 !important;
+      color: @text !important;
+    }
+
+    .button:active,
+    .button:focus,
+    .button:hover {
+      border-color: @subtext1 !important;
+      color: @text !important;
+    }
+
+    .button--primary,
+    .button--primary:active,
+    .button--primary:focus,
+    .button--primary:hover {
+      background-color: @accent-color !important;
+      border-color: @accent-color !important;
+      color: @base !important;
+      outline-color: @accent-color !important;
+    }
+
+    .button--primary:active,
+    .button--primary:focus {
+      border-color: @accent-color !important;
+    }
+
+    .danger {
+      color: @red !important;
+    }
+
+    .notification-bar {
+      background-color: @accent-color !important;
+      border-bottom-width: 0 !important;
+      color: @base !important;
+    }
+
+    .notification-bar--danger {
+      background-color: @red !important;
+    }
+
+    .notification-bar--success {
+      background-color: @green !important;
+    }
+
+    .notification-bar--banner .button {
+      color: @base !important;
+    }
+
+    .badge--warning {
+      background-color: @yellow !important;
+      border-color: @yellow !important;
+      color: @base !important;
+    }
+
+    .badge--warning:hover {
+      color: @base !important;
+    }
+
+    .badge--warning:active,
+    .badge--warning:focus {
+      outline-color: #00000000 !important;
```

Comment:
Unthemed?

Reply:
the outline color didn't add anything to the badge, if it was set to `@yellow` like the rest of the warning badges, it's unnoticable. if it's set to `@accent-color`, it looks ugly with the mismatched colors. same if its `@text`.

Reply:
the outline color didn't add anything to the badge, if it was set to `@yellow` like the rest of the warning badges, it's unnoticable. if it's set to `@accent-color`, it looks ugly with the mismatched colors. same if its `@text`.

Reply:
Is this the default color that PyPI has it set to?

Reply:
i kinda forgot that existed when writing that style, especially because vscode decided to stop recommending it as an autofill option for colors

Reply:
Ah okay, no worries!